src/pages/tracked-contracts/TrackedChain.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+interface TrackedChainProps {
+  name: string;
+  active: boolean;
+  onClick: () => void;
+}
+
+const TrackedChain: React.FC<TrackedChainProps> = ({ name, active, onClick }) => {
```

Comment:
Moved it here, in the same folder, I don't think it's gonna be used anywhere else so no need to put it inside components